refactor: PIN onboarding views

maxwellmattryan opened the pull request:
## Summary

> Please summarize your changes, describing __what__ they are and __why__ they were made.

As part of the onboarding changes, the two PIN setup views have been combined into one.

## Changelog

```
- Remove RepeatPin component
- Add confirm PIN input to SetupPin component
- Append "View" and "Router" to component names as necessary 
```

## Relevant Issues

> Please list any related issues using [development keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

Closes #3622

## Testing

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [x] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

- You **cannot** continue if...
    1. Either PIN code is in an invalid format
    2. The confirmation PIN input **does NOT** match the set PIN input 

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [x] I have made corresponding changes to the documentation

nicole-obrien reviewed (changes requested):
Can we make the following improvements to the UX:

- Autofocus on the first input
- Autofocus on the second input when the first is filled in
- Autofocus on button when second is filled in
- Increase gap between input box and subtitle

maxwellmattryan commented:
I mentioned in Discord, but I have added initial autofocus and fixed the input label placement. I have added the other UX improvements [here](https://github.com/iotaledger/firefly/issues/3627).
